src/lib.rs

```diff
@@ -340,7 +340,17 @@ mod hyper_impl {
             self: Pin<&mut Self>,
             cx: &mut Context<'_>,
         ) -> Poll<Option<Result<Self::Conn, Self::Error>>> {
-            self.poll_next(cx)
+            match self.poll_next(cx) {
+                Poll::Ready(Some(res)) => match res {
+                    Ok(val) => Poll::Ready(Some(Ok(val))),
+                    Err(e) => {
+                        println!("error {}", e);
```

Comment:
We probably shouldn't have a println here

Reply:
That makes sense. I'll remove that.